src/eslint.ts

```diff
@@ -47,6 +48,18 @@ export interface EslintOptions {
    * @default false
    */
   readonly prettier?: boolean;
+
+  /**
+   * Enable import alias for module paths
+   * @default undefined
+   */
+  readonly aliasMap?: [string, string][];
```

Comment:
Because we are using [jsii](https://aws.github.io/jsii/) under the hood to compile projen and generate APIs in other languages, I don't believe we can use certain fancier types like [string, string] in our exported interfaces. Instead we should probably change this to a record/dictionary type:
```suggestion
  readonly aliasMap?: { [key: string]: string };
```

...and add some code in the constructor that converts this back into an array of tuples.

Reply:
Ok, I'll make the necessary changes. I also considered automatically configuring this based on the `.tsconfig` that we have access to already within Eslint - but opted for this approach to be more explicit. If we need to deviate from the normal eslint structure it might be ok to have this implicit configuration based on aliases declared in the `.tsconfig`?

Reply:
Do you need to define import aliases in both places for source code to compile and lint without errors?

If the user provides import aliases/paths within the tsconfig options and we could pass them to the eslint config as well (so they don't have to write them twice), I think that could be convenient! I'm also okay with leaving them separate if needed.

Reply:
Yeah unfortunately need to define them in 3 places. Jest config, tsconfig and eslint. Let me try and work some magic to infer them from tsconfig as I think that would be a nicer developer experience (so long as it is documented what is happening)

Reply:
I ended up making the changes to the types only. The reference to tsconfig available was just the path, not the actual config options